### PR TITLE
updated sample build instructions

### PR DESCRIPTION
**Pull request by gherlein**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +56,14 @@ On Ubuntu and Raspberry Pi OS you can get the libraries by running
 $ sudo apt-get install libssl-dev libcurl4-openssl-dev liblog4cplus-dev libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev gstreamer1.0-plugins-base-apps gstreamer1.0-plugins-bad gstreamer1.0-plugins-good gstreamer1.0-plugins-ugly gstreamer1.0-tools
 ```
 
+#### Building the Sample Code
+
+The GStreamer and JNI examples are NOT built by default.  If you wish to build JNI you MUST add -DBUILD_JNI=TRUE and for GStreamer examples you MUST add -DBUILD_JNI=TRUE.  To build both:
```
Comment (disa6302) [inline README.md:61]: Can you modify the line to this: 

The GStreamer and JNI examples are NOT built by default.  If you wish to build JNI you MUST add `-DBUILD_JNI=TRUE` and for GStreamer plugin and samples, you MUST add `-DBUILD_GSTREAMER_PLUGIN=ON`

Comment (gherlein) [reply README.md]: Change made as new commit.  you should be able to do a merge squash when you accept it so I didn't squash it

Comment (disa6302) [reply README.md]: Not sure why, but I do not see the changes you made. The lines still are the same?

**Comment by gherlein** [comment]
I re-did the doc.  After looking at it again it needed cleanup.  Not sure what happened to the old changes but I just did it over clean to make your life easier.

**Pull request closed without merging**